pkg/sink/kafka/sarama.go

```diff
@@ -167,3 +184,68 @@ func completeSaramaSASLConfig(ctx context.Context, config *sarama.Config, o *Opt
 
 	return nil
 }
+
+func getKafkaVersion(ctx context.Context, config *sarama.Config, o *Options) (sarama.KafkaVersion, error) {
+	var err error
+	addrs := o.BrokerEndpoints
+	if len(addrs) > 1 {
+		// Shuffle the list of addresses to randomize the order in which
```

Comment:
Why shuffle here ?

Reply:
Prevent latency from increasing when the first address is inaccessible.